doc/examples/nonlinear_from_rest/submit_multiple_check.py

```diff
@@ -32,23 +43,39 @@
 
     Ra_numbers = np.logspace(np.log10(Ra_c_test), np.log10(1.04 * Ra_c_test), 4)
 
+    # Ra_numbers = [1710]
+    # aspect_ratio = 1.0
     for Ra in Ra_numbers:
 
         command = (
-            f"run_simul_check_from_python.py -R {Ra} -nx {nx} "
-            f"--order {order} --end-time {end_time} -np {nb_procs} "
-            f"-a_y {aspect_ratio} --stretch-factor {stretch_factor}"
+            f"run_simul_check_from_python.py -R {Ra} -Pr {prandtl} -nx {nx} "
+            f"--order {order} --dt-max {dt_max} --end-time {end_time} -np {nb_procs} "
+            f"-a_y {aspect_ratio} --stretch-factor {stretch_factor} "
+            f"--x-periodicity {x_periodicity} --y-periodicity {y_periodicity} "
```

Comment:
For arguments for boolean values, it's not useful to put the value. Just add `--x-periodicity` when needed.

Reply:
For example with something like:
```python
if x_periodicity:
    command += " --x-periodicity"
```